client/components/App.jsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react'
-import MessageList from './MessageList'
-
+// import MessageList from './MessageList'
```

Comment:
No, I only had it commented out so that because the routes weren't all set up yet and I wanted to be able to just get to my component with localhost:3000 in the meantime